conf/application.conf

```diff
@@ -204,9 +204,12 @@ microservice {
         tax-enrolments {
             host=localhost
             port=9802
-            serviceName5MLD="serviceName"
+            non-taxable {
+                serviceName="serviceName"
+                callback="http://localhost:9782/trusts/tax-enrolment/registration/non-taxable/hmrc-tersnt-org/:trn/subscriptions"
+            }
             serviceName="serviceName"
```

Comment:
Could this move into an object of it's own to match non-taxable?

Reply:
Done